arch/X86/X86GenAsmWriter1.inc

```diff
@@ -9,6 +9,8 @@
 /* Capstone Disassembly Engine, http://www.capstone-engine.org */
 /* By Nguyen Anh Quynh <aquynh@gmail.com>, 2013-2015 */
 
+#ifndef CAPSTONE_DIET
```

Comment:
Ugh, indeed, this is quite not nice. I don't really need cs_detail, but I am probably not alone here.

Among this code I could only remove printAliasInstr, which actually only does pretty printing of some instructions that printInstruction can also handle, but it won't really change the size that is taken by things like OpInfo. 

Do you have any ideas? Could revert the things and create a define that kills cs_detail for example.